server/routes/referrals/referralsController.test.ts

```diff
@@ -64,6 +69,40 @@ describe('POST /referrals', () => {
       expect(interventionsService.createDraftReferral).toHaveBeenCalledTimes(1)
     })
   })
+
+  describe('when a crn is not entered', () => {
+    it('reloads the page with a validation error', async () => {
```

Comment:
```suggestion
    it('renders a validation error', async () => {
```

Reply:
Also the invalid calls should probably return a 400 status, like the other controller actions here.